src/sql/util.py

```diff
@@ -250,56 +250,89 @@ def parse_toml_error(e, file_path):
         return e
 
 
-def get_user_configs(file_path):
+def get_user_configs(primary_path, alternate_path):
     """
     Returns saved configuration settings in a toml file from given file_path
 
     Parameters
     ----------
-    file_path : str
+    primary_path : Path
+        file path to a toml file
+    alternate_path : Path
         file path to a toml file
 
     Returns
     -------
     dict
         saved configuration settings
+    Path
+        the path of the file used to get user configurations
     """
-    data = load_toml(file_path)
-    section_names = ["tool", "jupysql", "SqlMagic"]
-    while section_names:
-        section_to_find, sections_from_user = section_names.pop(0), data.keys()
-        if section_to_find not in sections_from_user:
-            close_match = difflib.get_close_matches(section_to_find, sections_from_user)
-            if not close_match:
-                MESSAGE_PREFIX = (
-                    f"Tip: You may define configurations in "
-                    f"{file_path}. Please review our "
+    data = None
+    tip_displayed = False
```

Comment:
The `tip_displayed` and `disable_tip` logic looks confusing. Please add some comments

Reply:
Fixed [here](https://github.com/ploomber/jupysql/pull/935/files#diff-653c15e6161c97f356be04e072838ed15014af275c06019a7692ab727fa44b97R385-R386).

I also moved the message display logic to the end of the function so it should be much easier to understand now!